# Is there a hole under ANY of the sensors?
bool hole
# Details per sensor
# True = hole
# False = no hole
bool front_left
bool front_right
bool rear_right
bool rear_left
